Python Examples/TestMonitor/DeleteResults/delete_results.py

```diff
@@ -0,0 +1,147 @@
+"""
+SystemLink Enterprise TestMonitor delete results example
+
+This example creates a single test result and 
+deletes this created result by using delete result API and 
+creates multiple(five) test results and 
+deletes all these multiple results at once by using delete-results API.
+"""
+
+import uuid
+import sys
+import os
+import datetime
+from typing import Dict, List
+import click
+
+current = os.path.dirname(os.path.realpath(__file__))
+parent = os.path.dirname(current)
+sys.path.append(parent)
+
+import test_data_manager_client
+
+def get_test_result() -> Dict:
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+            "statusType": "RUNNING",
+            "statusName": "Running"
+        },
+        "systemId": None,
+        "hostName": None,
+        "properties":None,
+        "serialNumber": str(uuid.uuid4()),
+        "operator": "John Smith",
+        "partNumber": "NI-ABC-123-PWR1",
+        "fileIds":None,
+        "startedAt": str(datetime.datetime.utcnow()),
+        "totalTimeInSeconds": 0.0
+    }
+
+    return test_result
+
+
+def is_partial_success_response(response: Dict) -> bool:
+    return "error" in response.keys()
+
+
+def create_single_result() -> Dict:
+    test_result = get_test_result()
+
+    # create test result
+    response = test_data_manager_client.create_results(results=[test_result])
+    if is_partial_success_response(response):
+        raise Exception("Error occurred while creating new test result, please check for correct test result details and have correct access for creating the new test results")
+    test_result = response["results"][0]
+
+    return test_result
+
+
+def delete_single_result(result_id: str) -> None:
+    try:
+        test_data_manager_client.delete_result(result_id, True)
+        print(f"\nThe test result with Id = {result_id} has been deleted")
+    except:
+        print("Error occurred while deleting the test result, please check for the correct result Id and you have correct access for deleting the results")
+
+
+def create_and_delete_single_result() -> None:
+    
+    # create test result
+    print("Creating New test result")
+    test_result = create_single_result()    
+    print(f"Test result has been created under part number={test_result['partNumber']} with Id = {test_result['id']}")
+    
+    # delete test result
+    print("Press enter to delete the result")
+    input()
+    delete_single_result(test_result["id"])
+
+
+def create_multiple_results() -> List:
+    test_result = get_test_result()
+    result_ids = []
+    for i in range(0,5):
+        test_result['programName'] = test_result['programName'] + str(i)
```

Comment:
Is there any reason for creating results under unique program name?

And this results in appending the iteration number like this
![image](https://user-images.githubusercontent.com/94509966/233608923-c97cb424-63f7-4513-a314-04dea49fa80f.png)

Reply:
The reason for giving the unique program name is to show unique results in the application. I have updated the logic to append the iteration number to the default program name but not to the previous test result program name.